libtetris/src/lib.rs

```diff
@@ -56,6 +57,7 @@ impl<'de> serde::Deserialize<'de> for Controller {
                     hold: (v >> 5) & 1 != 0,
                     soft_drop: (v >> 6) & 1 != 0,
                     hard_drop: (v >> 7) & 1 != 0,
+                    rotate_180: (v >> 8) & 1 != 0,
```

Comment:
This one was a guess; I don't know if this change has any problems with backwards-compatibility and might require careful thought?